tools/WinObjC.Packaging/WinObjC.Packageable.targets

```diff
@@ -0,0 +1,88 @@
+    <HasConfigurationSpecificPackageContents Condition="'$(HasConfigurationSpecificPackageContents)' == ''">false</HasConfigurationSpecificPackageContents>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true'">build\lib\$(TargetOsAndVersion)\$(PlatformTarget)</PackageContentBasePath>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true' and '$(HasConfigurationSpecificPackageContents)' == 'true'">$(PackageContentBasePath)\$(Configuration)</PackageContentBasePath>
+
+    <ShouldAssignPackageHeaderBasePath Condition="'$(PackageHeaderBasePath)' == ''">true</ShouldAssignPackageHeaderBasePath>
+    <PackageHeaderBasePath Condition="'$(ShouldAssignPackageHeaderBasePath)' == 'true'">build\include\$(MSBuildProjectName)</PackageHeaderBasePath>
+
+    <ShouldAssignPackageResourcesBasePath Condition="'$(PackageResourcesBasePath)' == ''">true</ShouldAssignPackageResourcesBasePath>
+    <PackageResourcesBasePath Condition="'$(ShouldAssignPackageResourcesBasePath)' == 'true'">build\content\$(MSBuildProjectName)</PackageResourcesBasePath>
+
+    <ShouldAssignPackageBundleBasePath Condition="'$(PackageBundleBasePath)' == ''">true</ShouldAssignPackageBundleBasePath>
+    <PackageBundleBasePath Condition="'$(ShouldAssignPackageBundleBasePath)' == 'true'">build\bundle\$(MSBuildProjectName)</PackageBundleBasePath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets);FindPublicHeaders</GetPackageContentsDependsOn>
+  </PropertyGroup>
+
+  <!-- Add MSBuild outputs into the package -->
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents"  DependsOnTargets="$(GetPackageContentsDependsOn)" Condition="'$(IncludeDefaultPackageContents)' == 'true'">
+    <!-- Static Library Projects -->
+    <ItemGroup Condition="'$(ConfigurationType)' == 'StaticLibrary' And $(IncludeStaticLibraryInPackage)' == 'true'">
+      <!-- .lib -->
+      <_BuiltProjectOutputGroupStaticLib Include="@(BuiltProjectOutputGroupKeyOutput -> '%(FinalOutputPath)')" />
+      <PackageFile Include="@(_BuiltProjectOutputGroupStaticLib->Distinct())" Condition="Exists('%(FullPath)')">
+        <PackagePath>$(PackageContentBasePath)\%(Filename)%(Extension)</PackagePath>
+      </PackageFile>
+      <!-- Headers -->
+      <_IncludeFiles Include="@(PublicHeader -> '%(Source)')" />
+      <PackageFile Include="@(_IncludeFiles)" Condition="Exists('%(FullPath)')">
+        <PackagePath>$(PackageHeaderBasePath)\%(Filename)%(Extension)</PackagePath>
+      </PackageFile>
+      <!-- Resources -->
+      <PackageFile Include="@(SBResourceCopy->'%(FullPath)')" Condition="Exists('%(FullPath)')">
+        <PackagePath>$(PackageResourcesBasePath)\%(Filename)%(Extension)</PackagePath>
+      </PackageFile>
+    </ItemGroup>
+
+    <!-- WinRT Projects -->
```

Comment:
>WinRT [](start = 9, length = 5)

normal dlls could go through this path too and I think then you'd want to include the public headers here too.

The weird / hard part is going to be if its a WinRT component, we do want to include the generated WinRT headers instead of the normal headers (for C++\Cx consumptions since it doesn't technically go through the .winmd). 

We can use the winmd presense of an indicator to look for these though. I'd talk to muktesh / pradip about if / how objc2winmd produces these.